packages/document-service-api/src/controllers/documents.js

```diff
@@ -188,4 +190,40 @@ module.exports = {
       }
     },
   ],
+  deleteDocument: async (req, res) => {
+    const { documentId } = req.params;
+    const search = {
+      applicationId: req.params.applicationId,
+      id: documentId,
+    };
+
+    const doc = await Documents.findOne(search);
+
+    if (!doc) {
+      req.log.debug(search, 'Unknown document ID');
+
+      res.status(404).send({
+        message: 'Unknown document ID',
+      });
+      return;
+    }
+
+    try {
+      await Documents.deleteOne({ documentId });
```

Comment:
Does this work? The `documentId` in line 212 is `id` in line 197

Also, use the `search` object as that's got the index on.

I believe you could actually simplify the find/delete to a single call:

```
const { deletedCount } = await Documents.deleteOne(search)
if (deletedCount === 0) {
   req.log.debug(search, 'Unknown document ID');

    res.status(404).send({
      message: 'Unknown document ID',
    });
    return;
}
```